src/app/shared/palette/address/write-address-field.html

```diff
@@ -2,7 +2,7 @@
 
   <h3 class="heading-medium">{{caseField | ccdFieldLabel }}</h3>
 
-  <div class="form-group" id="postcodeLookup" [ngClass]="{'form-group-error': missingPostcode}">
+  <div class="form-group" id="postcodeLookup" [ngClass]="{'form-group-error': missingPostcode}" *ngIf="!isSearchField">
```

Comment:
can we rename this property to showPostcode? this way it's more generic. If in the future we have other sections of the ui where we don't want to show the postcode we wouldn't have to change the code again